modules/social_features/social_group/src/Plugin/views/field/SocialGroupViewsBulkOperationsBulkForm.php

```diff
@@ -197,7 +197,7 @@ public function viewsFormValidate(&$form, FormStateInterface $form_state) {
       // Grab all the actions that are available.
       foreach (Element::children($this->actions) as $action) {
         // If the option is not in our selected options, next.
-        if (($action_key = array_search($action, array_column($this->options['selected_actions'], 'action_id'))) === FALSE) {
+        if (($action_key = array_search($action, array_combine(array_keys($this->options['selected_actions']), array_column($this->options['selected_actions'], 'action_id')))) === FALSE) {
```

Comment:
```suggestion
        $selected_actions = array_combine(
          array_keys($this->options['selected_actions']),
          array_column($this->options['selected_actions'], 'action_id')
        );

        if (empty($selected_actions) ||
          ($action_key = array_search($action, $selected_actions)) === FALSE) {
```

Hello @tbsiqueira looks like I don't have access to update this PR so do you mind updating to above changes if this looks good to you or if you want we can close this PR and create new PR from my fork. Let me know thanks